### ISSUE TITLE
Duplicate search results for search and redact feature. 

### ISSUE THREAD

**Issue by Aman-Hundal** [bcgov/foi-flow#4670]
**Describe the bug in current situation**
There are duplicate results for searches being produced when using the Redaction Panel and searching a keyword in the docreivewer app. If you use the regular search you get the correct amount of results (notice the discrepancy in results found from redaction panel and regular search panel). These duplicate results in the redaction panel when clicked move the selection of the original results and when add mark is selected / redaction made only one redaction/annotation is created and nothing breaks (which is good and as expected). No errors are logged either. See screen shots below.

**Link bug to the User Story**
https://test-reviewer-web-marshal.apps.silver.devops.gov.bc.ca/foi/606

**Impact of this bug**
Low to medium impact as it does not break any critical features (ie. redaction)

**Chance of Occurring (high/medium/low/very low)**
High

**Pre Conditions: which Env, any pre-requesites or assumptions to execute steps?**
Tested on regular dev and test-marshal.

**Steps to Reproduce**
1. Go to a document request in the docreviewer (use the one linked above)
2. Click on the redaction panel
3. Search for a keyword
4. Notice that there are duplicates being retrieved in the results (ie. using the request example linked above, search "sample" and notice that results 3 and 6 are duplicates and 4 and 7 are duplicates"

**Actual/ observed behaviour/ results**
Duplicate instances of the search is retrieved and shown in the redaction panel search results.

**Expected behaviour**
One instance of the search result should be shown and retrieved in the redaction panel search results.

**Screenshots/ Visual Reference/ Source**
Search Result Image
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd1NHQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--cc7e11d8fbb8090f8ac39309cec64673682c1dee/image.png)

Redaction Panel Result Image with duplicates
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd2VHQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--983afe36a3f9923bc267f265173bb27b64518cb3/image.png)

**Comment by nimya-aot** [comment]
As a result of the above issue , need to fix this issue as well. 
When we search a single word on redaction panel,  same result comes up two times and if user accidently select both of them and click 'Add Mark' then two redactions are generating on the page. 
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOWVhQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--46f67af4996ca181b6d055321ad07b742f207935/MicrosoftTeams-image%20%281%29.png)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOWlhQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--017db4b59cf8daa6cfd6d31e1123e9458935964d/image.png)

**Comment by lmullane** [comment]
@Aman-Hundal , @nimya-aot , is the duplicate redaction box generated over the same word or is it redacting different content that the user had not selected?

cc @m-prodan

**Comment by nimya-aot** [comment]
@lmullane duplicate redaction box is over the same word. In the screenshot I separated two redaction boxes to show there are two redaction boxes.

**Comment by Aman-Hundal** [comment]
@lmullane Yep confirming what @nimya-aot  posted, it is redacting over the same word and the redaction boxes are on top of each other.

**Comment by Aman-Hundal** [comment]
Tested a file that had this duplicate issue on Apryses Demo here: https://showcase.apryse.com/redaction

Duplicate issue exists on Apryse's Demo as well. Will log a ticket with them as this appears to be a Apryse related issue.

**Comment by Aman-Hundal** [comment]
Ticket logged with Apryse - https://support.apryse.com/support/tickets/65393
